Source/ModelGroups/Mesh_PrimitiveMode.cs

```diff
@@ -1,488 +1,356 @@
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangles(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModePoints(properties, meshPrimitive);
+                    SetIndicesPoints(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeLines(properties, meshPrimitive);
+                    SetIndicesLines(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeLineLoop(properties, meshPrimitive);
+                    SetIndicesLineLoopOrTriangleFan(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeLineStrip(properties, meshPrimitive);
+                    SetIndicesLineStrip(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangleStrip(properties, meshPrimitive);
+                    SetIndicesTriangleStrip(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangleFan(properties, meshPrimitive);
+                    SetIndicesLineLoopOrTriangleFan(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangles(properties, meshPrimitive);
+                    SetIndicesTriangles(properties, meshPrimitive);
+                    SetIndicesComponentTypeInt(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangles(properties, meshPrimitive);
+                    SetIndicesTriangles(properties, meshPrimitive);
+                    SetIndicesComponentTypeByte(properties, meshPrimitive);
+                }),
+                CreateModel((properties, meshPrimitive) => {
+                    SetModeTriangles(properties, meshPrimitive);
+                    SetIndicesTriangles(properties, meshPrimitive);
+                    SetIndicesComponentTypeShort(properties, meshPrimitive);
+                }),
+            };
+
+            GenerateUsedPropertiesList();
         }
+    }
 
-        static internal class PointOnLine
+    static internal class PointOnLine
```

Comment:
Should be private and private usually goes before static.
```
private static class
```

Reply:
Fixed